### DISCUSSION TITLE
Issue with pgmq Extension Installation – Request for Assistance

### DISCUSSION THREAD

**Discussion by hotkaybee** [supabase/supabase#30091] [Questions] [answered]
I’ve encountered an issue while trying to add the pgmq extension via the dashboard. Unfortunately, it’s not listed among the available extensions. As a workaround, I attempted to install it using SQL with the following command:
`CREATE EXTENSION pgmq;`
However, this approach also failed, returning the following error:
```
ERROR: 0A000: extension "pgmq" is not available  
DETAIL: Could not open extension control file "/usr/share/postgresql/15/extension/pgmq.control": No such file or directory.
```
I’ve spent a considerable amount of time troubleshooting, referencing the documentation, and exploring possible solutions, but without success. The fact that pgmq is mentioned in the documentation yet unavailable is quite frustrating.
Could you please address this issue as soon as possible or provide an alternative solution? Your prompt assistance would be greatly appreciated.
![image](https://github.com/user-attachments/assets/61cbc830-c319-4701-be1a-6d141b94eb6d)
![image](https://github.com/user-attachments/assets/507a2415-eb4b-4684-86a7-99181a62c501)

**Comment by GaryAustin1** [comment, marked as answer]
Looks like this extension was added by Supabase 3 weeks ago to the Postgres image.
![image](https://github.com/user-attachments/assets/e36beae1-5b68-4f75-a129-49fe5a9ad7c2)

But that version is not release yet as near as I can tell.  Seems like they are on 15.6 right now and that is 15.8.  So you have to wait for that version to be released.

**Reply by GaryAustin1** [reply]
Progress can be tracked here...
https://github.com/supabase/postgres/releases
